test/v2/generators/DepartureMessageGeneratorSpec.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.generators
+
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+import v2.models.DepartureId
+import v2.models.MessageType
+import v2.models.MessageType.MRNAllocated
+
+import java.time.Clock
+import scala.xml.NodeSeq
+
+class DepartureMessageGeneratorSpec extends AnyFreeSpec with Matchers {
+
+  import Setup._
+  "A generator" - {
+    "when supplied with message type MRNAllocated" - {
+      "should produce an IE028 Message" - {
```

Comment:
`-` should be `in`, this isn't running the test you wrote.

Reply:
Agree.